config/karma/karma.conf.dev.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+var path = require("path");
+
+var MAIN_PATH = path.join(process.cwd(), "test/client/main.js");
+var POLYFILL_PATH = path.join(
+  path.dirname(require.resolve("core-js/package.json")), "es6/**/*.js"
+);
+var PREPROCESSORS = {};
+PREPROCESSORS[MAIN_PATH] = ["webpack"];
+PREPROCESSORS[POLYFILL_PATH] = ["webpack"];
+
+/*
+ * Karma Configuration: "dev" version.
+ *
+ * This configuration relies on a `webpack-dev-server` already running and
+ * bundling `webpack.config.test.js` on port 3001. If this is not running,
+ * then the alternate `karma.conf.js` file will _also_ run the webpack dev
+ * server during the test run.
+ */
+module.exports = function (config) {
+  config.set({
+    frameworks: ["mocha", "phantomjs-shim"],
```

Comment:
Suggest switching to headless chrome instead. Phantomjs is on it's way out and headless chrome is so much more reliable. (May solve your test-dev timeouts). 

For an example, this pr switched Radium to headless chrome (and other unrelated stuff); https://github.com/FormidableLabs/radium/pull/962/files